msgpack/__init__.py

```diff
@@ -3,7 +3,12 @@
 from .exceptions import *
 
 from collections import namedtuple
+from numbers import Number, Integral
 
+import struct
+import sys
+if sys.version_info[0] > 2:
+    long = int
```

Comment:
I will drop Python 2 support before adding timestamp support.
So no need to care about Python 2 compatibility.